src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,18 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    @Override
+    public String getPrintableString() {
+        return getPrintableString(this.getPhone(), this.getEmail(), this.getAddress());
+    }
+    
+    private String getPrintableString(Printable... printables){
+        String result = "";
+        for(Printable printable: printables){
+            result = printable.getPrintableString() + "\n";
+        }
+        return result;
+    }
```

Comment:
You can also check out how your teammates, put this in `TextUI.java`, #806 or #777